atoma-state/src/types.rs

```diff
@@ -247,31 +247,37 @@ pub struct Node {
 
 pub enum AtomaAtomaStateManagerEvent {
     /// Represents an update to the number of compute units in a stack
-    UpdateStackNumComputeUnits {
+    UpdateStackNumTokens {
         /// Unique small integer identifier for the stack
         stack_small_id: i64,
-        /// Estimated total number of compute units in the stack
-        estimated_total_compute_units: i64,
-        /// Total number of compute units in the stack
-        total_compute_units: i64,
+        /// Estimated total number of tokens in the stack
+        estimated_total_tokens: i64,
+        /// Total number of tokens in the stack
+        total_tokens: i64,
         /// Number of concurrent requests for the stack
         concurrent_requests: u64,
     },
     /// Represents an update to the amount of fiat currency for a user to be locked
     LockFiatAmount {
         /// User address
         user_address: String,
-        /// Amount in fiat currency
-        amount: i64,
+        /// Amount to be locked in fiat currency for input
+        estimated_input_amount: i64,
+        /// Amount to be locked in fiat currency for output
+        estimated_output_amount: i64,
     },
     /// Represents an update to the amount of fiat currency for a user, after the inference is done
     UpdateFiatAmount {
         /// User address
         user_address: String,
-        /// Estimated total amount in fiat currency
-        estimated_total_amount: i64,
-        /// Total amount in fiat currency
-        total_amount: i64,
+        /// Estimated total amount for input in fiat currency
+        estimated_input_amount: i64,
+        /// Total amount for input in fiat currency
+        input_amount: i64,
+        /// Estimated total amount for output in fiat currency
+        estimated_output_amount: i64,
+        /// Total amount for output in fiat currency
+        output_amount: i64,
     },
     /// Represents an update to the total hash of a stack
     UpdateStackTotalHash {
```

Comment:
Are we still using this ?

Reply:
on dev branch there is still use for that apparently
anyway it's unrelated to this PR

Reply:
We need to get rid of this asap, as your latest changes regarding update total hash introduced a major issue with disk storage